### ISSUE TITLE
Charm only grants flight when out of Beacon range

### ISSUE THREAD

**Issue by LessThanBenji** [LemmaEOF/ShulkerCharm#11]
Hi, I really hope this is a bug and I haven't misunderstood the intended behavior of the mod. I'm using v1.2.0 on 1.20.1 on Fabric 0.15.7. When in range of a Beacon with the Charm equipped, the player is granted a Slow Falling buff that renews endlessly whenever the player isn't in contact with the ground (ie. jumping or falling). However, Creative flight is not granted at all -- until the player moves out of Beacon range. Only then does the Charm grants Creative flight, and the Slow Falling buff stops renewing until the Charm's power runs out, as intended.

The charging behavior of the Charm is working normally. I haven't touched the config file, I'm using all the default values.

Is the Charm not supposed to grant flight whilst in range of the Beacon? Or is this a bug?

Happy to explain further or provide logs, if needed. I am using a swathe of other mods alongside Shulker Charm, but I can't think of any that could be interfering. I'll test Shulker Charm on its own shortly, just to rule that out.

**Comment by LessThanBenji** [comment]
False alarm! It works in a standalone environment. I'll work out which mod was the culprit and post it here in case anyone else comes across the same issue.

**Comment by LessThanBenji** [comment]
Figured it out -- it's Trinkets, strangely. Trinkets v3.7.1 works, but Trinkets 3.7.2 causes the broken behavior I described above.

I installed Shulker Charm via the Modrinth app where it automatically chose and loaded Trinkets v3.7.1, which works. Soon as I pressed the Modrinth app's "Update all" button, Trinkets got bumped up to v3.7.2 and Shulker Charm broke.

I'm not sure if this means this is a Shulker Charm issue or a Trinkets issue. If it's a problem on their end and not yours, I'll happily create an issue on their issue tracker -- I've checked and there doesn't seem to be one currently.

TL;DR: If you're having the same issue as me, downgrade Trinkets to v3.7.1.

**Comment by milkev** [comment]
theres no changelog for trinkets 3.7.2 (why trinkets) but ill try and look into it this afternoon

**Comment by milkev** [comment]
Fixed in both new releases for 1.20.1 and 1.20.2-1.20.4

**Comment by LessThanBenji** [comment]
Works like a charm ;) nice job!
